### PR TITLE
update maven instructions for beta

### PR DESCRIPTION
**Pull request by mbroz2**
#2769

### CODE REVIEW

File: templates/beta-release-post.adoc [hunk 1]
```diff
@@ -103,21 +103,47 @@ To enable the new beta features in your app, add them to your `server.xml`:
...
 
-If you're using link:{url-prefix}/guides/maven-intro.html[Maven], here are the coordinates:
+If you're using link:{url-prefix}/guides/maven-intro.html[Maven], you can install the All Beta Features package using:
 
+[source,xml]
+----
+<plugin>
+    <groupId>io.openliberty.tools</groupId>
+    <artifactId>liberty-maven-plugin</artifactId>
+    <version>3.3.4</version>
```
Comment (scottkurz) [inline templates/beta-release-post.adoc]: 3.3.4 => 3.7.1  
(might as well do the latest as of now)

File: templates/beta-release-post.adoc [hunk 2]
```diff
@@ -103,21 +103,47 @@ To enable the new beta features in your app, add them to your `server.xml`:
...
+    <configuration>
+        <runtimeArtifact>
+          <groupId>io.openliberty.beta</groupId>
+          <artifactId>openliberty-runtime</artifactId>
+          <version>RELEASE_VERSION</version>
```
Comment (scottkurz) [inline templates/beta-release-post.adoc:125]: I think the `<version>RELEASE_VERSION</version` could make it too easy for someone to replace with "22.0.0.12" which wouldn't work.

We could do: 
  `<version>YY.0.0.MM-beta</version>`

or even just  
 `<version>LATEST</version>`

**Review by scottkurz — CHANGES_REQUESTED** [review]

File: templates/beta-release-post.adoc (hunk 2 above)
Comment (mbroz2) [reply templates/beta-release-post.adoc:125]: This a template file for the beta release blog post.  There's a GH Action that replaces RELEASE_VERSION throughout the file with the passed in release, like 23.0.0.1-beta

File: templates/beta-release-post.adoc (hunk 1 above)
Comment (mbroz2) [reply templates/beta-release-post.adoc]: maybe we should do something like:
```suggestion
    <version>[3.7.1,)</version>
```

File: templates/beta-release-post.adoc [hunk 3]
```diff
@@ -103,21 +103,47 @@ To enable the new beta features in your app, add them to your `server.xml`:
...
 
-If you're using link:{url-prefix}/guides/maven-intro.html[Maven], here are the coordinates:
+If you're using link:{url-prefix}/guides/maven-intro.html[Maven], you can install the All Beta Features package using:
 
+[source,xml]
+----
+<plugin>
+    <groupId>io.openliberty.tools</groupId>
+    <artifactId>liberty-maven-plugin</artifactId>
+    <version>[3.7.1,)</version>
```
Comment (mbroz2) [inline templates/beta-release-post.adoc]:
```suggestion
    <version>RELEASE</version>
```

**Review by scottkurz — APPROVED** [review]

**Pull request merged**